go.mod

```diff
@@ -56,6 +56,7 @@ require (
 	github.com/joho/godotenv v1.5.1
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/lensesio/tableprinter v0.0.0-20201125135848-89e81fc956e7
+	github.com/lib/pq v1.10.9
```

Comment:
This dependency isn't used – try running `go mod tidy`

Reply:
using it to get rid of the custom Tags type.